src/components/Button/Button.tsx

```diff
@@ -35,19 +37,21 @@ interface BaseButtonProps {
 }
 
 // Button props
-type ButtonElementProps = React.ButtonHTMLAttributes<HTMLButtonElement> &
+export type ButtonElementProps = React.ButtonHTMLAttributes<HTMLButtonElement> &
```

Comment:
I switched `Dropdown.Menu.Item` to use `Button` components, and exported this for reuse over there.